CRM/Activity/BAO/Activity.php

```diff
@@ -1699,16 +1699,7 @@ public static function addActivity(
     $params = []
   ) {
     $date = date('YmdHis');
-    if ($activity->__table == 'civicrm_membership') {
-      $component = 'Membership';
-    }
-    elseif ($activity->__table == 'civicrm_participant') {
-      if ($activityType != 'Email') {
-        $activityType = 'Event Registration';
```

Comment:
activityType is used lower down on line 1732

Reply:
opps - yes I didn't mean to remove that!